components/Dialogs/src/MDCAlertController.h

```diff
@@ -177,6 +177,13 @@
 /** Descriptive text that summarizes a decision in a sentence of two. */
 @property(nonatomic, nullable, copy) NSString *message;
 
+/** 
+ Descriptive attributed text that summarizes a decision in a sentence of two. 
+
+ If provided and non-empty, will be used instead of @c message property.
+ */
+@property(nonatomic, nullable, copy) NSAttributedString *attributedMessage;
+
```

Comment:
```suggestion
/**
 Descriptive attributed text that summarizes a decision in a sentence of two.

 If provided and non-empty, will be used instead of @c message property.
 */
@property(nonatomic, nullable, copy) NSAttributedString *attributedMessage;

```

Reply:
Removing trailing whitespace.